package.json

```
@@ -21,18 +21,26 @@
     "bignumber.js": "^9.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.19.3",
+    "@babel/core": "^7.19.6",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.19.4",
+    "@babel/preset-typescript": "^7.18.6",
+    "@babel/register": "^7.18.9",
+    "@goodeggs/toolkit": "^7.0.1",
+    "@goodeggs/tsconfig": "^2.0.2",
     "chai": "^4.2.0",
-    "coffee-script": ">=1.8.x",
     "mocha": "^7.1.0"
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "always-auth": true
   },
   "scripts": {
-    "compile": "coffee --bare --compile --output lib/ src/",
-    "prepublish": "npm run compile",
-    "pretest": "npm run compile",
-    "test": "mocha"
+    "test": "mocha",
```

The `test` command should run all unit tests and/or integration tests and/or linting and/or typechecking. See https://github.com/goodeggs/standards-and-best-practices/blob/2e4a4782bea91954abf3c3323c57f9dd2d8ca15f/javascript/package-scripts.md#all-repos.
```suggestion
    "test": "yarn run lint && yarn run test:unit",
    "test:unit": "mocha",
```
Don't forget to add `&& yarn run typecheck` when you port to TS in the next PR.